### DISCUSSION TITLE
UI UX designer

### DISCUSSION THREAD

**Discussion by tezalaee** [WeAreMahsaAmini/Core#377] [Introductions]
Hey I am designer from Rome,
I could help out on projects if needed

**Comment by 3nity-Design** [comment]
Hey! I'm a UX designer & researcher interested in collaborating! How can I kick off an initiative?
By Far I've designed the attached posters didn't know where exactly they should go. 
![WLL - Front](https://user-images.githubusercontent.com/114813091/193410324-b8a092a6-b9f7-42d7-a43e-a007589f0aeb.jpg)
![WLL - Back](https://user-images.githubusercontent.com/114813091/193410327-8039b927-9736-4b49-92ad-57002534d820.png)

**Reply by 3nity-Design** [reply]
I'm based in Vancouver, Canada.
